cluster/src/main/java/org/apache/iotdb/cluster/server/heartbeat/HeartbeatThread.java

```diff
@@ -86,15 +84,17 @@ public void run() {
             // send heartbeats to the followers
             sendHeartbeats();
             synchronized (localMember.getHeartBeatWaitObject()) {
-              localMember.getHeartBeatWaitObject().wait(RaftServer.getHeartBeatIntervalMs());
+              localMember.getHeartBeatWaitObject().wait(RaftServer.getHeartbeatIntervalMs());
             }
             hasHadLeader = true;
             break;
           case FOLLOWER:
             // check if heartbeat times out
-            long heartBeatInterval =
+            long heartbeatInterval =
                 System.currentTimeMillis() - localMember.getLastHeartbeatReceivedTime();
-            if (heartBeatInterval >= RaftServer.getConnectionTimeoutInMS()) {
+            long randomElectionTimeout =
+                RaftServer.getElectionTimeoutMs() + getElectionRandomTimeOutMs();
```

Comment:
In fact, I don't think we need to calculate a randomElectionTimeout every time we receive a heartbeat, we just need to save a calculated randomElectionTimeout for every term. 
But as it's trivil, I can accept this way for now.

Reply:
OK, this is only a call of a random number.